### ISSUE TITLE
[UX] 10-10EZR: Design Review/Subtask flow

### ISSUE THREAD

**Issue by alexseelig** [department-of-veterans-affairs/va.gov-team#68381]
As a Veteran, I would like to see what information VHA has for me on file and edit it accordingly so that I don't have to go through lots of screens just to see the thing I want to edit.

- [x] Review the wireframe workflows created back in September (ish?)
- [x] Reference any other applications that have done this (only one I know of is Check In Experience)
- [x] Design the "Review" page in a way that the Veteran can see all of their EZR information
- [x] Design the "Review" page to account for TERA, EC, NoK, and Military History
- [x] Design the "subtask" workflow where Veteran can edit each of the different fields independent of other sections.
- [x] Determine what information needs to be required in a subtask section (or if it's the same as what is currently required)
- [x] Design the save and save in progress process
- [x] Design this for the sort form (AS Note: If this is a big lift we can break into separate story)
- [x] Demo to team and talk through architecture
- [x] Make tweaks as needed
- [x] **Success: We have Review/Subtask designs that have been reviewed by the entire team and are ready to be expanded upon (if needed) or implemented**

**Comment by jessicastump** [comment]
@alexseelig Clarification on this ticket too please? Is this for the v2 design work once all the prefill BE work is ready?

**Comment by IMHieuVo** [comment]
Blocked - A huge part of the EZR that the mockup struggled to address is how to effectively surface the financial information (which is the primary reason for EZR). 

Things to consider:
- It should not be pre-filled because the amount may be different from year to year.
- Can we surface the previous amount as a reference?
- How are we handling multiple dependent and multiple financial information? 

We'll explore a choose-your-own adventure flow to see if we can learn anything on it while this problem marinates.

**Comment by IMHieuVo** [comment]
Additional exploration for Choose-your-own-adventure #77911 

Choose-Your-Own-Adventure:
PRO: 
- Flexibility*: allows the user to update only the information they came to update (*still have to review all the information within the form)
- Personalized: This route sets up for future personalized recommendations and resources
CON:
- Complex development: UX/Dev needs to consider branching paths, conditional logic,  and various outcome
- Custom Pattern: the Accordion pattern is not customizable, this would be an experimental pattern starting from scratch.
- Information overload: on the first screen.
- Content management: can be tricky with the number of possible paths.
- Collab Cycle: new pattern may need to go through the Collab Cycle.
- Navigation: Navigation can be frustrating/ confusing if user is forced to back back to hub or review information they were not prompted to update.

Confirmation Flow:
PRO:
- Skip Collab Cycle: developing an existing experimental pattern.
- Clarity and control: each section (card) functions independently and be developed independently.
- Clarity and control: linear flower allows a clear and structured process that is easy to follow and understand.
- Reduce cognitive load: presenting information and action in a linear fashion is beneficial for Veterans with cognitive challenges.
CON:
- Everything is custom.
- Laborious development: Not a form library pattern, Dev needs to manually validate inputs in every section.
- Limited flexibility: Users may be frustrated by the constraint path, especially for those that only want to update some information.

Based on the discovery and exploration, the Confirmation flow is still the most efficient flow throw the form. We'll move forward with the Confirmation Flow and tackle it in sections.
